code/modules/jobs/job_types/roguetown/adventurer/types/wretch/deserter.dm

```diff
@@ -240,6 +240,880 @@
+				var/weapons = list("Steel Knuckles","Axe","Sword","Whip","Spear")
+				var/weapon_choice = input("Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+				switch(weapon_choice)
+					if("Steel Knuckles")
+						beltr = /obj/item/rogueweapon/knuckles
+					if("Axe")
+						beltr = /obj/item/rogueweapon/stoneaxe/boneaxe
+					if("Sword")
+						beltr = /obj/item/rogueweapon/sword/short
+					if("Whip")
+						beltr = /obj/item/rogueweapon/whip
+					if("Spear")
+						r_hand = /obj/item/rogueweapon/spear/bonespear
+
+			if("Hide Armor Kini")
+				armor = /obj/item/clothing/suit/roguetown/armor/leather/hide/bikini
+				pants = /obj/item/clothing/under/roguetown/heavy_leather_pants/shorts
+				wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+				shoes = /obj/item/clothing/shoes/roguetown/boots/furlinedboots
+				gloves = /obj/item/clothing/gloves/roguetown/fingerless_leather
+				backl = /obj/item/storage/backpack/rogue/satchel
+				belt = /obj/item/storage/belt/rogue/leather
+				neck = /obj/item/storage/belt/rogue/pouch/coins/poor
+				backpack_contents = list(/obj/item/flashlight/flare/torch = 1, /obj/item/rogueweapon/huntingknife = 1)
+				var/weapons = list("Steel Knuckles","Axe","Sword","Whip","Spear","MY BARE HANDS!!!")
+				var/weapon_choice = input("Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+				switch(weapon_choice)
+					if("Steel Knuckles")
+						beltr = /obj/item/rogueweapon/knuckles
+					if("Axe")
+						beltr = /obj/item/rogueweapon/stoneaxe/boneaxe
+					if("Sword")
+						beltr = /obj/item/rogueweapon/sword/short
+					if("Whip")
+						beltr = /obj/item/rogueweapon/whip
+					if("Spear")
+						r_hand = /obj/item/rogueweapon/spear/bonespear
+			
+			if("Studded Leather Kini")
+				armor = /obj/item/clothing/suit/roguetown/armor/leather/studded/bikini
+				pants = /obj/item/clothing/under/roguetown/tights/black
+				wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+				shoes = /obj/item/clothing/shoes/roguetown/boots
+				gloves = /obj/item/clothing/gloves/roguetown/fingerless_leather
+				backl = /obj/item/storage/backpack/rogue/satchel
+				belt = /obj/item/storage/belt/rogue/leather
+				neck = /obj/item/storage/belt/rogue/pouch/coins/poor
+				backpack_contents = list(/obj/item/flashlight/flare/torch = 1, /obj/item/rogueweapon/huntingknife/idagger/steel = 1)
+				var/weapons = list("Katar","Rapier","Whip","Billhook")
+				var/weapon_choice = input("Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+				switch(weapon_choice)
+					if ("Katar")
+						beltr = /obj/item/rogueweapon/katar
+					if("Rapier")
+						beltr = /obj/item/rogueweapon/sword/rapier
+					if("Whip")
+						beltr = /obj/item/rogueweapon/whip
+					if("Billhook")
+						r_hand = /obj/item/rogueweapon/spear/billhook
+
+
+	H.adjust_blindness(-3)
+	H.change_stat("speed", 3) /// Not a fighter, even with a virtue, built to be able to escape from fights.
```

Comment:
ditto - you have given them an absurd skill block and dodge expert, and three speed and constitution